Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -33,20 +35,108 @@ of requiring self adjointness in the definition.
 Positive operator
 -/
 
-
-open InnerProductSpace RCLike ContinuousLinearMap
+open InnerProductSpace RCLike LinearMap ContinuousLinearMap
 
 open scoped InnerProduct ComplexConjugate
 
-namespace ContinuousLinearMap
-
 variable {𝕜 E F : Type*} [RCLike 𝕜]
 variable [NormedAddCommGroup E] [NormedAddCommGroup F]
 variable [InnerProductSpace 𝕜 E] [InnerProductSpace 𝕜 F]
-variable [CompleteSpace E] [CompleteSpace F]
 
 local notation "⟪" x ", " y "⟫" => @inner 𝕜 _ _ x y
 
+namespace LinearMap
+
+variable [FiniteDimensional 𝕜 E] [FiniteDimensional 𝕜 F]
+
+/-- A linear map `T` of a Hilbert space is **positive** if it is self adjoint and
+  `∀ x, 0 ≤ re ⟪T x, x⟫`. -/
+def IsPositive (T : E →ₗ[𝕜] E) : Prop :=
+  IsSelfAdjoint T ∧ ∀ x, 0 ≤ re ⟪T x, x⟫
```

Comment:
I think you should prove (under the `ComplexOrder` scope), that this is equivalent to `IsSelfAdjoint T ∧ ∀ x, 0 ≤ ⟪T x, x⟫`.